src/libsyntax/util/lev_distance.rs

```diff
@@ -52,14 +52,16 @@ where
     T: Iterator<Item = &'a Symbol>,
 {
     let max_dist = dist.map_or_else(|| cmp::max(lookup.len(), 3) / 3, |d| d);
+    let name_vec: Vec<&Symbol> = iter_names.collect();
```

Comment:
Hmm I'm not sure how to do that. If we call `cloned()`, `iter_names` is moved and we can't use that iterator again, and we still can't use the cloned one twice. Is there another way to clone it so that we can still use the original `iter_names` afterwards?